packages/http-client/src/ceramic-http-client.ts

```diff
@@ -92,7 +93,9 @@ export class CeramicClient implements CeramicApi {
 
     this.pin = new RemotePinApi(this._apiUrl)
     this.index = new RemoteIndexApi(this._apiUrl)
-    this.admin = new RemoteAdminApi(this._apiUrl)
+    this.admin = new RemoteAdminApi(this._apiUrl, () => {
+      return this.did
```

Comment:
Is `this.did` really in scope on this anon function?

Reply:
ooph good call.  This isn't actually tested at the moment.  Was going to add tests in a follow-up, but now I feel like it might be worth doing before making the RC.  Will try to put something together right now

Reply:
Actually, with my basic testing, it seemed to work as-written.  But I added `.bind(this)` to the end just to be safe anyway